dotnet/src/SemanticKernel/Diagnostics/NullableAttributes.cs

```diff
@@ -1,6 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This one needs to stay. We should see if we can stop the tools from flagging it.

Reply:
Brought it back, but it's keeping me from passing dotnet-format